dsn.go

```diff
@@ -515,6 +521,38 @@ func fillMissingConfigParameters(cfg *Config) error {
 	return nil
 }
 
+func extractDomainFromHost(host string) (domain string, index int) {
+	i := strings.Index(strings.ToLower(host), topLevelDomainPrefix)
+	if i >= 1 {
+		domain = host[i:]
+		return domain, i
+	}
+	return "", i
+}
+
+func extractDomainFromRegion(region string) string {
```

Comment:
Possibly change the name, extract suggests  that domain is a substring of region

Reply:
Done